### PR TITLE
(chore) Factor out environment variables

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Factors out [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables) for our  [turborepo-gh-artifacts](https://github.com/felixmosh/turborepo-gh-artifacts)  into an `env` block scoped to the `build` job description.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,10 +47,10 @@ jobs:
         run: yarn run test
       
       - name: Run lint and typecheck
-        run: yarn turbo run lint typescript --color --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="${{ github.repository_owner }}"
+        run: yarn turbo run lint typescript --color"
```
Comment (ibacher) [inline .github/workflows/ci.yml:50]:
```suggestion
        run: yarn turbo run lint typescript --color
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -43,10 +47,10 @@ jobs:
...
       
       - name: Run build
-        run: yarn turbo run build --color --concurrency=5 --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="${{ github.repository_owner }}"
+        run: yarn turbo run build --color --concurrency=5"
```
Comment (ibacher) [inline .github/workflows/ci.yml:53]:
```suggestion
        run: yarn turbo run build --color --concurrency=5
```

**Comment by denniskigen** [comment]
Thanks, @ibacher!

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks!

**Pull request merged**